### ISSUE TITLE
Verify Podman installation onboarding workflow

### ISSUE THREAD

**Issue by odockal** [containers/podman-desktop#4672]
Podman onboarding workflow requires to be covered by E2E tests. 

Edit: We can test a hybrid scenario without actually spinning up podman msi with GUI. During the onboarding, we could come to a point where user is being asked if he wants to install podman, search for podman msi and trigger the installation in the background. then proceed with onboarding. Requires investigation.

**Comment by cbr7** [comment]
@odockal I'm not usre I understand what this issue is about, what should be added to the current podman onboarding test suite for this?

**Comment by odockal** [comment]
@cbr7 This e2e test is meant to cover the scenario where we go through global onboarding (one available from welcome page), on podman section we check the podman installation workflow. It requires goax framework to be used to operate the native msi/pkg installer on particular platform.

**Comment by odockal** [comment]
@cbr7 We might consider closing the issue as not planned since we will have new dashboard soon. WDYT? then new issue can be opened and addressed unless it is already being worked on.

**Comment by cbr7** [comment]
Yes, I agree with you, I'm going to mark it as not planned and then we can deal with it when the new dashboard is implemented fully in 1.29.
